drivers/media/i2c/irs1125.c

```diff
@@ -82,6 +82,7 @@ struct irs1125 {
 	struct v4l2_ctrl *ctrl_numseq;
 
 	int power_count;
+	bool ctrl_init;
```

Comment:
I can't comment on the commit text directly, but the line
> Thus, this ctrl is not written inside the driver into after power up. The userspace has to take care to write senseful data here.

is a little oddly phrased.
>Thus, this ctrl is not written to the hardware until it has been initialised by userspace. Userspace must take care to write meaningful data to the control - it is not validated.

Secondly ```ctrl_init``` is rather generic. It's specifically pll_ctrl_init, or pll_ctrl_set.

Reply:
I hope, the comment is better now.

I renamed `ctrl_init` to `mod_pll_init`